client/components/data/query-experiments/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External Dependencies
+ */
+
+import { useEffect, FunctionComponent } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { fetchExperiments } from 'state/experiments/actions';
+import { getAnonId, nextRefresh } from 'state/experiments/selectors';
+import { AppState } from 'types';
+
+type QueryProps = {
+	doFetchExperiments: typeof fetchExperiments;
+	updateAfter: number;
+	anonId: string;
+};
+
+const QueryExperiments: FunctionComponent< QueryProps > = ( {
```

Comment:
IMO, Nope, not worth it. The query component uses effects, a connected component, and is really simple. Testing it would require reaching deep into React to verify that the effect was there and executed correctly.

In theory, you might be tempted to break the if-statement from the effect into a function that we then test, but that's deceptively simple since it works off a closure from the props, which comes from the sate. So, in practice, it's a lot of work to test a dead-simple if-statement and `fetchExperiments` is already tested elsewhere.

I didn't look very hard, but I didn't see any other query components with tests, with the exception of some very complex ones.